godot-core/src/builtin/plane.rs

```diff
@@ -217,7 +217,14 @@ impl Plane {
     /// Returns a normalized copy of the plane.
```

Comment:
I know this isn't part of the changeset, but that doc line is really not helpful, as it contains exactly the information of the method and nothing more.

Since you also fixed the implementation, could you maybe describe more precisely what "normalization" in this context means?